cve_bin_tool/parsers/ruby.py

```diff
@@ -1,4 +1,6 @@
-# Copyright (C) 2022 Intel Corporation
+# NOTE: Done
```

Comment:
```suggestion
# Copyright (C) 2024 Intel Corporation
```

Looks like this was just an accident, but we should keep the copyright line here and might as well update the year.